src/blueapi/cli/cli.py

```diff
@@ -172,8 +173,13 @@ def run_plan(
     client: BlueapiRestClient = obj["rest_client"]
 
     logger = logging.getLogger(__name__)
-
-    amq_client = AmqClient(StompMessagingTemplate.autoconfigured(config.stomp))
+    _message_template = StompMessagingTemplate.autoconfigured(config.stomp)
+    if _message_template is not None:
+        amq_client = AmqClient(_message_template)
+    else:
+        raise RuntimeError(
+            "Message bus needs to be configured to get done message after run"
+        )
```

Comment:
```suggestion
        raise RuntimeError(
            "Cannot run plans without Stomp configuration to track progress"
        )
```

Reply:
It still might be worth changing this to a custom exception to confirm that the tests are hitting this code, as a `RuntimeError` could be raised by something else.

Reply:
That is logical, I have added the test for it